src/styles/reset.scss

```
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
```

I think it is useful to also include those rules:
```suggestion
 margin: 0;
 padding: 0;
```